gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -225,4 +225,9 @@ public Completable insertOrUpdateTile(Tile tile) {
   public Maybe<Tile> getTile(String tileId) {
     return db.tileDao().findById(tileId).map(TileEntity::toTile);
   }
+
+  @Override
+  public Maybe<Integer> deleteTile(String tileId) {
+    return db.tileDao().findById(tileId).flatMap(tileEntity -> db.tileDao().deleteTile(tileEntity));
+  }
```

Comment:
Do we care about the number of tiles deleted? If so, why `Maybe`? And if not, we should just return a Completable?